accelerator/apis.c

```diff
@@ -280,7 +280,7 @@ status_t api_find_transactions_by_tag(const iota_client_service_t* const service
   find_transactions_res_t* res = find_transactions_res_new();
   if (req == NULL || res == NULL) {
     ret = SC_TA_OOM;
-    log_error(apis_logger_id, "[%s:%d]\n", __func__, __LINE__);
+    ta_log_error("\n");
```

Comment:
We should print the error status code (or the error code name) here

Reply:
BTW, I prefer there is at least one argument in the `ta_log_error()`
e.g. `ta_log_error("%s\n", "error code here");`
Nothing but `\n` looks strange for me